docs/literate/src/files/getting_started_with_Trixi.jl

```diff
@@ -0,0 +1,287 @@
+#src # Getting started with Trixi.jl
+
+# This tutorial is intended for beginners in Julia and Trixi.jl.
+# After reading it, you will install Julia and Trixi on your PC and will be able to download files
+# from Trixi github, execute and edit them.
+# **Trixi.jl** is a numerical simulation framework for hyperbolic conservation laws 
+# written in [`Julia`](https://julialang.org/).
+# This means that Julia have to be installed on a PC to work with Trixi. 
+
+# ## Julia installation
+
+# Trixi works with the current stable Julia release. More information about Julia support can be
+# found in [`README`](https://github.com/trixi-framework/Trixi.jl#installation).
+# The most fully explained installation process can be found in
+# [`Julia installation instruction`](https://julialang.org/downloads/platform/).
+# But you can follow also our short installation instruction.
+
+# ### Windows
+
+# - Download Julia [`installer`](https://julialang.org/downloads/) for Windows. Make sure 
+#   that you choose the right version of installer (64-bit or 32-bit) according to your computer.
+# - Open the downloaded installer.
+# - Paste an installation directory path or find it using a file manager (select `Browse`).
+# - Select `Next`.
+# - Check the `Add Julia to PATH` to add Julia to the Environment Variables. 
+#   This makes possible to run Julia using Terminal from any directory only typing `julia`.
+# - Select `Next`, then Julia will be insalled.
+
+# Now you can verify, that Julia is installed:
+# - Type `Win+R` on a keyboard.
+# - Enter `cmd` in opened window.
+# - Enter in a terminal `julia`. 
+
+# Then Julia will be invoked. To close Julia enter `exit()`. 
+
+# ### Linux
+
+# - Open a Terminal and navigate (using `cd`) to the directory, where you want to save Julia.
+#   Or you can open a file manager, find this directory, right-click inside and 
+#   choose `Open Terminal Here`.
+# - To install Julia execute the following commands in a Terminal:
+#   ````
+#   wget https://julialang-s3.julialang.org/bin/linux/x64/1.8/julia-1.8.5-linux-x86_64.tar.gz
+#   tar zxvf julia-1.8.5-linux-x86_64.tar.gz
+#   ````
+#   Now you can verify that Julia is installed entering `julia` command in a Terminal.
+
+# Then Julia will be invoked. To close Julia enter `exit()`.
+
+# ## Trixi installation
+
+# Trixi and its related tools are registered Julia packages. So installation of them is
+# running inside Julia. For appropriate work of Trixi you need to install 
+# [`Trixi`](https://github.com/trixi-framework/Trixi.jl),
+# [`Trixi2Vtk`](https://github.com/trixi-framework/Trixi2Vtk.jl),
+# [`OrdinaryDiffEq`](https://github.com/SciML/OrdinaryDiffEq.jl) and 
+# [`Plots`](https://github.com/JuliaPlots/Plots.jl).
+
+# - Open a Terminal.
+# - Invoke Julia executing `julia`.
```

Comment:
Again, this only works at the moment for Windows, since you have not explained how to add `julia` to the PATH on Linux/macOS. You don't have to, but maybe you can then modify this line accordingly?

Reply:
I added in the part with installing Julia on Linux a link to official explanation of Julia-team, how to add `julia` to PATH. Without adding I have to change `julia` into `path/to/<Julia directory>` in part with cloning Trixi.jl, that will disable possibility to just copy and paste commands in a terminal. Alternatively, I can explain exactly the way, how to setup `julia` to the PATH on Linux, without referencing to another guide. But problem with MacOs remains, I can only copy commands or guides from other authors from internet, but without guarantee of course, that it will work. (I've no MacOs at hand)

Reply:
Have you tried [juliaup](https://github.com/JuliaLang/juliaup)? Maybe this could be an alternative to explain how to install Julia, as it seems to set things like PATH automatically. It will install the latest Julia release and you can always provide a link to the manual downloadable files as a fallback.